polaris-core/src/main/resources/schemas/policies/system/metadata-compaction/2025-02-03.json

```diff
@@ -0,0 +1,37 @@
+{
+  "license": "Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)",
+  "$id": "https://polaris.apache.org/schemas/policies/system/metadata-compaction/2025-02-03.json",
```

Comment:
Does the JSON magically land at the location specified in the ID somehow? Or do we always need a followup PR?

Also, it looks a little funny to use dates here given that the date in the PR may not align with the date the schema actually becomes effective. In the worst case, we could merge two versions in one day. Maybe just an incrementing number is easier?

Reply:
Unfortunately no. I hope I can publish it once for all based on the directory structure.

I want to keep them as the same for date as these are first batch, should be fine as nobody is using it. Once we release them at 1.0. We should follow the date schema strictly.